src/resource_clients/task.ts

```diff
@@ -55,8 +55,8 @@ export class TaskClient extends ResourceClient {
      * Starts a task and immediately returns the Run object.
      * https://docs.apify.com/api/v2#/reference/actor-tasks/run-collection/run-task
      */
-    async start(input: Dictionary | Dictionary[], options: TaskStartOptions = {}): Promise<ActorRun> {
-        ow(input, ow.optional.object);
+    async start(input: string | Dictionary, options: TaskStartOptions = {}): Promise<ActorRun> {
+        ow(input, ow.optional.any(ow.object, ow.string));
```

Comment:
But it needs to be a JSON string. That's why we kept object, to make sure that it's JSON serializable before it hits API.

Reply:
Could validate that the string is a valid JSON string? Would that be better, or should we just stick to allowing just objects?

Reply:
Yeah, could be. I guess I was just lazy when I was making those validations. Thinking that the number of users who would want to use JSON string as an input instead of an object would be really low.

Reply:
What do you suggest I do? make a validator that makes sure the string is a valid json object, or leave it as is?

Reply:
How does it behave now? I guess in case of invalid JSON there will be some error when we call `JSON.parse` anyway? Which seems enough to me, so I would keep it as is in such case.

If we would validate this, we should not parse the JSON string twice (once for validation and once for actual payload) to mitigate possible performance issues.

Reply:
Where do we call `JSON.parse` exactly? It seems we only do that for the response, not the request...unless axios does it by default?

Reply:
I would just keep it as object. We can allow string if some users request, but it's super easy for them to parse their own strings into objects, whereas for us, it's a bit of a pain to do the checks. And also I'm pretty sure that if we allow type string, people will start putting random strings there.